_docs_integrate/integration-example.md

```diff
@@ -290,7 +272,12 @@ Then, tap on "New Message", enter a subject and body and tap on "Send".
 In order to fetch the Message, we need to synchronize the Connector with the Backbone again.
 
 {% include rapidoc api_route_regex="^post /api/v2/Account/Sync$" %}
-The Response should contain a Message with the `content` you entered in the App.
+
+After syncing, all Messages can be displayed with the `GET /api/v2/Messages` route.
+
+{% include rapidoc api_route_regex="^get /api/v2/Messages$" %}
+
+The response should contain a Message with the `content` you entered in the App.
```

Comment:
we should also talk about that there will be an event that you can subscribe to. Take the entry to events as soon and as often as possible